src/work-allocation/release2/components/available-tasks-filter/available-tasks-filter.component.spec.ts

```diff
@@ -0,0 +1,155 @@
+import { Location } from '@angular/common';
+import { Component, ViewChild } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { Router } from '@angular/router';
+import { ExuiCommonLibModule } from '@hmcts/rpx-xui-common-lib';
+import { of } from 'rxjs';
+
+import * as dtos from 'src/work-allocation/models/dtos';
+import { LocationDataService } from 'src/work-allocation/services';
+import { getMockLocations } from 'src/work-allocation/tests/utils.spec';
+import { FilterConstants } from 'src/work-allocation/components/constants';
+import { AvailableTasksFilterRelease2Component } from './available-tasks-filter.component';
+
+@Component({
+  template: `<exui-available-tasks-filter-v2 (selectionChanged)="onSelectionChanged($event)"></exui-available-tasks-filter-v2>`
```

Comment:
Can you do something other than shove a `-v2` suffix on the end?

Reply:
changed it to -r2. seems more appropriate and the tech leads seem happy with it